components/script_plugins/utils.rs

```diff
@@ -30,12 +30,8 @@ pub fn match_def_path(cx: &LateContext, def_id: DefId, path: &[Symbol]) -> bool
 }
 
 pub fn in_derive_expn(span: Span) -> bool {
-    if let Some(i) = span.ctxt().outer_expn().expn_info() {
-        if let ExpnKind::Macro(MacroKind::Attr, n) = i.kind {
-            n.as_str().contains("derive")
-        } else {
-            false
-        }
+    if let ExpnKind::Macro(MacroKind::Attr, n) = span.ctxt().outer_expn_data().kind {
+        n.as_str().contains("derive")
```

Comment:
I don’t know if this is stil correct, since https://doc.rust-lang.org/nightly/nightly-rustc/syntax_pos/hygiene/enum.MacroKind.html#variant.Derive is (now?) a thing. Our libscript still compiles, though.